### PR TITLE
Fix: Point to start.vaadin.com

### PR DESCRIPTION
**Pull request by tarekoraby**
start.vaadin.com (not vaadin.com/start) is the preferred way to start a new Vaadin app.

### CODE REVIEW

File: scripts/generator/templates/template-release-notes-maintenance.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Here are the highlighted new and improved features in this maintenance release.
 
 # Getting Started with Vaadin 14
 ## App starters
-The best way to get started with Vaadin 14 is to go to [https://vaadin.com/start](https://vaadin.com/start) and pick an app template for the technology stack you’re interested in.
+The best way to get started with Vaadin 14 is to go to [https://start.vaadin.com](https://start.vaadin.com) and pick an app template for the technology stack you’re interested in.
```
Comment (web-padawan) [inline scripts/generator/templates/template-release-notes-maintenance.md]: > the technology stack you’re interested in.

We should rephrase this part as there isn't an option to choose a stack at https://start.vaadin.com (only Spring Boot).

**Review by web-padawan — CHANGES_REQUESTED** [review]

File: scripts/generator/templates/template-release-notes-maintenance.md (hunk 1 above)
Comment (tarekoraby) [reply scripts/generator/templates/template-release-notes-maintenance.md]: @web-padawan I already did while you wrote your review 😄

Comment (web-padawan) [reply scripts/generator/templates/template-release-notes-maintenance.md]: Thanks!

**Review by ZheSun88 — APPROVED** [review]

**Review by ZheSun88 — APPROVED** [review]

**Comment by ZheSun88** [comment]
i will pick those manually.. dont worry

**Pull request merged**